src/main/java/com/dev/test/config/AppConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.dev.test.config;
+
+import com.dev.test.model.User;
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@ComponentScan(basePackages = {
+        "com.dev.test.service",
+        "com.dev.test.dao"
+})
+public class AppConfig {
+    @Bean
+    public DataSource getDataSource() {
+        BasicDataSource dataSource = new BasicDataSource();
+        dataSource.setDriverClassName("com.mysql.cj.jdbc.Driver");
+        dataSource.setUrl("jdbc:mysql://localhost/spring_test?serverTimezone=UTC");
+        dataSource.setUsername("java");
+        dataSource.setPassword("Qwertyuiop1!");
+        return dataSource;
+    }
+
+    @Bean
+    LocalSessionFactoryBean getSessionFactory() {
```

Comment:
```suggestion
    private LocalSessionFactoryBean getSessionFactory() {
```

Reply:
it can't be private/
set it public
done